package.json

```diff
@@ -28,7 +28,8 @@
     "refapp3PatientSearch": "cypress run --spec src/test/resources/features/refapp-3.x/06-patient-search/patient-search.feature",
     "refapp3Settings": "cypress run --spec src/test/resources/features/refapp-3.x/03-settings/settings.feature",
     "refapp3ClinicalVisit": "cypress run --spec src/test/resources/features/refapp-3.x/04-clinical-visit/clinical-visit.feature",
-    "refapp3VitalsAndTriage": "cypress run --spec src/test/resources/features/refapp-3.x/05-vitals-and-triage/vitals-and-triage.feature"
+    "refapp3VitalsAndTriage": "cypress run --spec src/test/resources/features/refapp-3.x/05-vitals-and-triage/vitals-and-triage.feature",
+    "refappOclSubscriptionModule": "cypress run --spec src/test/resources/features/oclsubscriptionmodule/index.feature"
```

Comment:
which refapp version the test against?

Reply:
It's the UI that's part of the module... It's in both 2.x and 3.x for now.